container_templates/python/container_example/README.md

```diff
@@ -0,0 +1,56 @@
+ # Container Example Code
+
+This the basic code for creating a Docker container for the Level 2 pipeline. 
+For questions contact: laura.sandoval@lasp.colorado.edu or contact Laura Sandoval via Slack.
+
+## Python Setup
+
+It is recommended that a virtual environment is created within the project:
+1. Make a virtual environment using `python -m venv venv` and activate it 
+with `source venv/bin/activate`. Python 3.10 is required.
```

Comment:
```suggestion
with `source venv/bin/activate`. Python 3.10 is required.
```
I don't think we are making any restrictions on Python version if they are within a container? If we are, then you should also update the base image to be from python3.10 instead of 3.9.